.github/workflows/mobile-android_build.yml

```diff
@@ -187,22 +207,23 @@ jobs:
     # Return to using:
     #   ./bazelw mobile-install --fat_apk_cpu=x86_64 --start_app //examples/kotlin/hello_world:hello_envoy_kt
     # When https://github.com/envoyproxy/envoy-mobile/issues/853 is fixed.
-    - name: 'Start kotlin app'
+    - name: 'Start Kotlin app'
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       run: |
         cd mobile
         ./bazelw build \
-            --config=mobile-remote-ci-macos \
+            --config=mobile-remote-release-clang \
             --fat_apk_cpu=x86_64 \
+            --linkopt=-fuse-ld=lld \
             //test/kotlin/apps/baseline:hello_envoy_kt
-        adb install -r --no-incremental bazel-bin/test/kotlin/apps/baseline/hello_envoy_kt.apk
-        adb shell am start -n io.envoyproxy.envoymobile.helloenvoybaselinetest/.MainActivity
+        "${ANDROID_HOME}/platform-tools/adb" install -r --no-incremental bazel-bin/test/kotlin/apps/baseline/hello_envoy_kt.apk
```

Comment:
nit: Do we need quotes here?

Would it make sense to put "${ANDROID_HOME}/platform-tools/" into PATH so we don't have to repeat the full path to adb.

Reply:
We don't need the full quotes, but we definitely need to quote a variable, i.e. `"${ANDROID_HOME}"`. It's a good practice to do that. See https://www.shellcheck.net/wiki/SC2086

As for updating the `PATH`, typically we don't want to do that CI. Being explicit is always better IMO.

I updated the PR to only quote `"${ANDROID_HOME}"`, which is also what I typically do.

Reply:
Good point!